Sources/Tracking/CustomerIO.swift

```diff
@@ -20,7 +24,7 @@ public protocol CustomerIOInstance: AutoMockable {
         // sourcery:TypeCast="AnyEncodable(body)"
         body: RequestBody
     )
-
+    func registeredDeviceToken() -> String?
```

Comment:
```suggestion
    var registeredDeviceToken: String? { get }
```

As mentioned in a previous PR, it would be ideal if this was a property instead of a function call.